test2.java

```diff
@@ -0,0 +1,9 @@
+class HelloWorld {
+    public static void main(String[] args) {
+        String stripeSensitiveInfo = "sk_test_kovrMB0mupFJXfNZWx6Etg5y"
+		String githubSensitiveInfo = "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"
+		String awsAccessKeyId = "AKIA6RIEUTC2YH6NTI5N"
+		String awsAccessSecretKey = "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"
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error, insert &#34;;&#34; to complete LocalVariableDeclarationStatement